OpenRA.Mods.Common/Widgets/ProductionTabsWidget.cs

```diff
@@ -233,6 +233,7 @@ public void ActorChanged(Actor a)
 		{
 			if (a.Info.HasTraitInfo<ProductionQueueInfo>())
 			{
+				// TODO: Should this extend to controlling allied units?
```

Comment:
In my option this is a question beyond this PR. I would remove the comment from this PR.